composer.json

```diff
@@ -15,28 +15,28 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
+        "php": ">=8.0.2",
```

Comment:
i know symfony itself does this, but i strongly oppose. we have no clue what php 9 will be like, and chances are reasonable that there will be BC breaks. if we don't limit to the major version, this code might be happily installed by composer in a php 9 application and not work as expected.

Reply:
Ah of course! That's been updated to ^8.0 (the Symfony Framework requirements will ensure this is >=8.0.2)